### PR TITLE
Block editor: Alt+F10 shouldn't scroll to top

### PR DESCRIPTION
**Pull request by ellatrix**
## Description

#19769 has caused a small regression: if you press Alt+F10 to focus the block toolbar, the page scroll to the top. This is because the toolbar now has an absolute position instead of a fixed position. At the time that the toolbar receives focus, the position of the toolbar is not set yet, and it will be invisible at the top of the page. When the browser places the focus, it also scroll the page within view, but shortly after that we position the popover, so it ends up being out of view.

One solution could be to set the position earlier, but this is tricky because for some popovers, we need the height, which is not available early enough.

I went for leaving the position to be fixed until we can set the popover position. Since it's initially fixed, the page won't scroll since the element is within the viewport.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

File: packages/components/src/popover/index.js [hunk 1]
```diff
@@ -246,6 +246,7 @@ const Popover = ( {
 	__unstableSlotName = SLOT_NAME,
 	__unstableAllowVerticalSubpixelPosition,
 	__unstableAllowHorizontalSubpixelPosition,
+	__unstableFixedPosition = true,
```
Comment (youknowriad) [inline packages/components/src/popover/index.js]: I'm a bit concerned about the growing number of unstable props for this component. This makes me think there's an audit work that's needed to rethink/consolidate these props. It becomes very hard to understand how this component works.

The "modifiers" approach `popper.js` uses could absorb a lot of these I think.

Comment (ellatrix) [reply packages/components/src/popover/index.js]: Yes, it's concerning, but I don't see an easy way around it for now. These unstable props allow us to experiment with the popover without committing to an API.

* `__unstableSticky` can probably be stabilised at some point.
* `__unstableSlotName` should ideally go away.
* `__unstableAllowVerticalSubpixelPosition` and `__unstableAllowHorizontalSubpixelPosition` when Chrome fixes the positioning bug (they're working on it), or if we can find a different workaround.
* `__unstableFixedPosition` could be stabilised eventually.

Comment (aduth) [reply packages/components/src/popover/index.js]: Minor: Noting that "could be stabilized" is meant to be a differentiator between what we consider to be "experimental" vs. "unstable".

>An unstable API is one which serves as a means to an end. It is not desired to ever be converted into a public API.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis

**Comment by ellatrix** [comment]
It looks like this also happens when tabbing in navigation mode. :(

**Comment by ellatrix** [comment]
(So this should definitely be merged for next release.)

File: packages/components/src/popover/index.js (hunk 1 above)
Comment (ellatrix) [reply packages/components/src/popover/index.js]: They all do serve as means to an end. If it's interesting for other popovers, we can consider making it experimental or stable though. I think we should avoid marking anything stable or experimental if we're thinking about using a third party library.

**Review by youknowriad — APPROVED** [review]
It fixes the issue for me.

**Comment by ellatrix** [comment]
Thanks! I'll be looking more into popover at some point. I'm sorry about the `__unstable` for now.

**Pull request merged**